Sources/TuistKit/Generator/ManifestTargetGenerator.swift

```diff
@@ -0,0 +1,48 @@
+import Basic
+import Foundation
+import TuistCore
+
+protocol ManifestTargetGenerating {
+    func generateManifestTarget(for project: String, at path: AbsolutePath) throws -> Target
+}
+
+class ManifestTargetGenerator: ManifestTargetGenerating {
+    private let manifestLoader: GraphManifestLoading
+    private let resourceLocator: ResourceLocating
+
+    init(manifestLoader: GraphManifestLoading,
+         resourceLocator: ResourceLocating) {
+        self.manifestLoader = manifestLoader
+        self.resourceLocator = resourceLocator
+    }
+
+    func generateManifestTarget(for project: String,
+                                at path: AbsolutePath) throws -> Target {
+        let settings = Settings(base: try manifestTargetBuildSettings(),
+                                debug: nil,
+                                release: nil)
+        let manifest = try manifestLoader.manifestPath(at: path, manifest: .project)
+        return Target(name: "\(project)-Manifest",
+                      platform: .macOS,
+                      product: .framework,
+                      bundleId: "",
```

Comment:
Do we need to set a bundle identifier ?

Reply:
I double checked what SwiftPM does for the package description target - it leaves it unset (i.e. `nil`) - might need to make this optional too at this level, I opted for blank to minimise disruptions to the API - perhaps an alternate could be `io.tuist.manifest.\(project)` or even `io.tuist.manifest.${PRODUCT_NAME:rfc1034identifier}`